### ISSUE TITLE
RT-DETR FP16 inference get correct result on v100 but weird result on a10

### ISSUE THREAD

**Issue by xiaochus** [NVIDIA/TensorRT#3652]
## Description

I am using tritonserver:23.10 to deploy RT-DETR model. The onnxruntime fp32, onnxruntime fp16 and Tesla V100 TRT 8.6.1  FP16/F32 both get the correct result. But Tesla A10 TRT 8.6.1 get correct result in FP32 and weird result in FP16. The FP16 result should be same both in V100 and A10 with the same code.

A10 - TRT - FP16 
![image](https://github.com/NVIDIA/TensorRT/assets/16798444/54b5590a-4548-492e-a061-b4e4508f0d8d)

V100 - TRT - FP16 
![image](https://github.com/NVIDIA/TensorRT/assets/16798444/4a8a64ba-d7f9-498b-82d6-810292776f26)

## Environment

**TensorRT Version**: TensoRT 8.6.1

**NVIDIA GPU**: Tesla V100 / A10

**NVIDIA Driver Version**: 515.65.01 

**CUDA Version**: 12.2

**CUDNN Version**: v8

Operating System: Ubuntu 22.04

Python Version (if applicable): 3.10

Tensorflow Version (if applicable):

PyTorch Version (if applicable): 2.0.1

Baremetal or Container (if so, version):  tritonserver 23.10

## Relevant Files

https://github.com/lyuwenyu/RT-DETR

## Steps To Reproduce

/usr/src/tensorrt/bin/trtexec --onnx=model.onnx --saveEngine=model.plan --fp16

**Comment by zerollzeng** [comment]
1. Did you test metric like mAP?
2. Could you please share the onnx here for reproduce.

Thanks!

**Comment by zerollzeng** [comment]
FP16 may introduce accuracy drop so it's hard to said whether it's bug unless we have generic metric like mAP.

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, thanks all!

**Comment by chinakook** [comment]
https://github.com/NVIDIA/TensorRT/issues/3700

**Comment by chinakook** [comment]
Please reopen this issue to track Ampere accuracy lose issue on **all detr like** models.
